README.md

```diff
@@ -4,59 +4,75 @@
 [![Coveralls Status](https://img.shields.io/coveralls/FAForever/server/develop.svg)](https://coveralls.io/github/FAForever/server)
 [![semver](https://img.shields.io/badge/license-GPLv3-blue)](license.txt)
 
-This is the source code for the [Forged Alliance Forever](http://www.faforever.com/) server.
+This is the source code for the
+[Forged Alliance Forever](https://www.faforever.com/) lobby server.
 
 # Contributing
 
-To contribute, please fork this repository and make pull requests to the develop branch.
-
-Use the normal git conventions for commit messages, with the following rules:
- - Subject line shorter than 80 characters
- - Proper capitalized sentence as subject line, with no trailing period
- - For non-trivial commits, always include a commit message body, describing the change in detail
- - If there are related issues, reference them in the commit message footer
+Before opening a pull request, please take a moment to look over the
+[contributing guidelines](CONTRIBUTING.md).
 
 ## Setting up for development
 
-First make sure you have an instance of [faf-db](https://github.com/FAForever/db) setup. Then install the dependencies to a virtual environment
-using pipenv by running:
+First, follow the instructions on the [faf-db repo](https://github.com/FAForever/db)
+to setup an instance of the database. Then install the pinned versions of the
+dependencies (and dev dependencies) to a virtual environment using pipenv by
+running:
 
     $ pipenv sync --dev
 
 You can then start the server in development mode with:
 
     $ pipenv run devserver
 
-You will see some exception trace when the server fails to connect to RabbitMQ,
+You will see an exception trace when the server fails to connect to RabbitMQ,
```

Comment:
Another error to ignore (during tests mostly?) is 
```
ERROR    LobbyConnection:lobbyconnection.py:580 Failure updating NickServ password for test
```
I think I wasted 30min on that at the start.

Reply:
Added a list of errors that can be ignored